### PR TITLE
Optimize description to production deployment using tiup

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Optimize description to production deployment using tiup

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://internal.pingcap.net/jira/browse/DOCS-206

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -278,50 +248,65 @@ alertmanager_servers:
...
 > **注意：**
 >
 > 通过 TiUP 进行集群部署可以使用密钥或者交互密码方式来进行安全认证：
->
-> - 如果是密钥方式，可以通过 `-i` 或者 `--identity_file` 来指定密钥的路径；
-> - 如果是密码方式，可以通过 `-p` 进入密码交互窗口；
+
+> - 如果是密钥方式，可以通过 -i 或者 --identity_file 来指定密钥的路径；
+
+> - 如果是密码方式，可以通过 -p 进入密码交互窗口；
```
Comment (shichun-0415) [inline production-deployment-using-tiup.md:264]:
```suggestion
> - 如果是密钥方式，可以通过 -i 或者 --identity_file 来指定密钥的路径。

> - 如果是密码方式，可以通过 -p 进入密码交互窗口。
```

**Comment by TomShawn** [comment]
@shichun-0415 Please fix the Jenkins failure. Thanks~

File: production-deployment-using-tiup.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
 
 在中控机上安装 TiUP 组件有两种方式：在线部署和离线部署。
 
+<SimpleTab>
+<div label="在线部署 TiUP 组件">
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:23]:
```suggestion
<div label="在线部署">
```

File: production-deployment-using-tiup.md [hunk 3]
```diff
@@ -75,6 +78,11 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
     tiup --binary cluster
     ```
 
+</div>
+
+<SimpleTab>
+<div label="离线部署 TiUP 组件">
+
 ### 方式二：离线部署 TiUP 组件
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:86]:
```suggestion
### 离线部署
```

File: production-deployment-using-tiup.md [hunk 4]
```diff
@@ -19,6 +19,9 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
 
 在中控机上安装 TiUP 组件有两种方式：在线部署和离线部署。
 
+<SimpleTab>
+<div label="在线部署 TiUP 组件">
+
 ### 方式一：在线部署 TiUP 组件
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:25]:
```suggestion
### 在线部署 TiUP 组件
```

File: production-deployment-using-tiup.md [hunk 5]
```diff
@@ -75,6 +78,11 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
     tiup --binary cluster
     ```
 
+</div>
+
+<SimpleTab>
+<div label="离线部署 TiUP 组件">
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:84]:
```suggestion
<div label="离线部署">
```

File: production-deployment-using-tiup.md [hunk 6]
```diff
@@ -19,6 +19,9 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
 
 在中控机上安装 TiUP 组件有两种方式：在线部署和离线部署。
 
+<SimpleTab>
+<div label="在线部署 TiUP 组件">
+
 ### 方式一：在线部署 TiUP 组件
 
 使用普通用户登录中控机，以 `tidb` 用户为例，后续安装 TiUP 及集群管理操作均通过该用户完成：
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:27]:
```suggestion
以普通用户身份登录中控机。以 `tidb` 用户为例，后续安装 TiUP 及集群管理操作均通过该用户完成：
```

File: production-deployment-using-tiup.md [hunk 7]
```diff
@@ -176,9 +184,11 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
         ```bash
         tiup mirror merge tiup-custom-mirror-v1.7.0
         ```
-    
+
     5. 上述步骤完成后，通过 `tiup list` 命令检查执行结果。在本文例子中，使用 `tiup list tiup` 和 `tiup list cluster` 均应能看到对应组件的 `v1.7.0` 版本出现在结果中。
 
+</div>
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:190]: 放这里好像不对。应该把下面 #### 部署离线环境 TiUP 组件  也包到这个 div 里吧？

File: production-deployment-using-tiup.md [hunk 8]
```diff
@@ -197,78 +207,48 @@ source /home/tidb/.bash_profile
...
+        - Grafana servers
+        - ......
+
+    下表列出了常用的 7 种场景，请根据链接中的拓扑说明以及配置文件模板新建一个配置文件`topology.yaml`。如果有其他组合场景的需求，请根据多个模板自行调整。
+
+    | 场景 | 配置文件模板 | 拓扑说明 |
+    | :-- | :-- | :-- |
+    | [最小拓扑架构](/minimal-deployment-topology.md) | [简单最小配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-mini.yaml)<br/>[详细最小配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-mini.yaml) | 最基本的集群拓扑，包括 tidb-server、tikv-server、pd-server，适合 OLTP 业务。 |
+    | [增加 TiFlash 拓扑架构](/tiflash-deployment-topology.md) | [简单 TiFlash 配置模版](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-tiflash.yaml)<br/>[详细 TiFlash 配置模版](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-tiflash.yaml) | 在最小拓扑的基础上部署 TiFlash。TiFlash 是列式存储引擎，已经逐步成为集群拓扑的标配。适合 Real-Time HTAP 业务。 |
+    | [增加 TiCDC 拓扑架构](/ticdc-deployment-topology.md) | [简单 TiCDC 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-cdc.yaml)<br/>[详细 TiCDC 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-cdc.yaml) | 在最小拓扑的基础上部署 TiCDC。TiCDC 是 4.0 版本引入的 TiDB 增量数据同步工具，支持多种下游 (TiDB/MySQL/MQ)。 |
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:247]:
```suggestion
    | [增加 TiCDC 拓扑架构](/ticdc-deployment-topology.md) | [简单 TiCDC 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-cdc.yaml)<br/>[详细 TiCDC 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-cdc.yaml) | 在最小拓扑的基础上部署 TiCDC。[TiCDC](/ticdc/ticdc-overview.md) 是 4.0 版本引入的 TiDB 增量数据同步工具，支持多种下游 (TiDB/MySQL/MQ)。 |
```

File: production-deployment-using-tiup.md [hunk 9]
```diff
@@ -197,78 +207,48 @@ source /home/tidb/.bash_profile
...
+    | [增加 TiDB Binlog 拓扑架构](/tidb-binlog-deployment-topology.md) | [简单 TiDB Binlog 配置模板（下游为 MySQL）](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-tidb-binlog.yaml)<br/>[简单 TiDB Binlog 配置模板（下游为 file）](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-file-binlog.yaml)<br/>[详细 TiDB Binlog 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-tidb-binlog.yaml) | 在最小拓扑的基础上部署 TiDB Binlog。TiDB Binlog 是目前广泛使用的增量同步组件，可提供准实时备份和同步功能。 |
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:248]:
```suggestion
    | [增加 TiDB Binlog 拓扑架构](/tidb-binlog-deployment-topology.md) | [简单 TiDB Binlog 配置模板（下游为 MySQL）](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-tidb-binlog.yaml)<br/>[简单 TiDB Binlog 配置模板（下游为 file）](https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-file-binlog.yaml)<br/>[详细 TiDB Binlog 配置模板](https://github.com/pingcap/docs-cn/blob/master/config-templates/complex-tidb-binlog.yaml) | 在最小拓扑的基础上部署 TiDB Binlog。[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 是目前广泛使用的增量同步组件，可提供准实时备份和同步功能。 |
```

File: production-deployment-using-tiup.md [hunk 10]
```diff
@@ -278,50 +258,64 @@ alertmanager_servers:
...
+
+    ```shell
+    tiup cluster deploy tidb-test v5.3.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+    ```
 
-预期日志结尾输出会有 ```Deployed cluster `tidb-test` successfully``` 关键词，表示部署成功。
+以上部署示例中：
+
+- `tidb-test` 为部署的集群名称。
+- `v5.2.2` 为部署的集群版本，可以通过执行 `tiup list tidb` 来查看 TiUP 支持的最新可用版本，后续内容以版本 `v5.2.2` 为例。
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:313]:
```suggestion
- `v5.3.0` 为部署的集群版本，可以通过执行 `tiup list tidb` 来查看 TiUP 支持的最新可用版本。
```
后面没有版本相关的内容了。所以删掉了那句话。

File: production-deployment-using-tiup.md [hunk 11]
```diff
@@ -333,13 +327,6 @@ tiup cluster list
 
 TiUP 支持管理多个 TiDB 集群，该命令会输出当前通过 TiUP cluster 管理的所有集群信息，包括集群名称、部署用户、版本、密钥信息等：
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:328]:
```suggestion
TiUP 支持管理多个 TiDB 集群，该命令会输出当前通过 TiUP cluster 管理的所有集群信息，包括集群名称、部署用户、版本、密钥信息等。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: production-deployment-using-tiup.md (hunk 7 above)
Comment (hfxsd) [inline production-deployment-using-tiup.md:190]:
```suggestion
</div>

</simpletab>
```

File: production-deployment-using-tiup.md [hunk 12]
```diff
@@ -75,6 +78,11 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
     tiup --binary cluster
     ```
 
+</div>
+
+<SimpleTab>
```
Comment (hfxsd) [inline production-deployment-using-tiup.md:83]:
```suggestion
```
simpletab 标签只在开始和结束的地方加。中间都用 div 分割。

File: production-deployment-using-tiup.md (hunk 7 above)
Comment (shichun-0415) [reply production-deployment-using-tiup.md]: Fixed

File: production-deployment-using-tiup.md [hunk 13]
```diff
@@ -195,80 +202,53 @@ source /home/tidb/.bash_profile
...
-tiup cluster template > topology.yaml
-```
-
-> **注意：**
->
-> 混合部署场景也可以使用 `tiup cluster template --full > topology.yaml` 生成的建议拓扑模板，跨机房部署场景可以使用 `tiup cluster template --multi-dc > topology.yaml` 生成的建议拓扑模板。
-
-执行 `vi topology.yaml`，查看配置文件的内容：
+</div>
+<SimpleTab>
```
Comment (shichun-0415) [inline production-deployment-using-tiup.md:206]:
```suggestion
</SimpleTab>
```

File: production-deployment-using-tiup.md [hunk 14]
```diff
@@ -195,80 +202,53 @@ source /home/tidb/.bash_profile
...
-
-- [增加 TiDB Binlog 拓扑架构](/tidb-binlog-deployment-topology.md)
-
-    包含最小拓扑的基础上，同时部署 TiDB Binlog。TiDB Binlog 是目前广泛使用的增量同步组件，可提供准实时备份和同步功能。
-
-- [增加 TiSpark 拓扑架构](/tispark-deployment-topology.md)
+## 第 3 步：初始化集群拓扑文件
 
-    包含最小拓扑的基础上，同时部署 TiSpark 组件。TiSpark 是 PingCAP 为解决用户复杂 OLAP 需求而推出的产品。TiUP cluster 组件对 TiSpark 的支持目前为实验性特性。
+1. 生成集群拓扑文件
```
Comment (qiancai) [inline production-deployment-using-tiup.md:210]:
```suggestion
1. 创建集群拓扑文件。
```

File: production-deployment-using-tiup.md [hunk 15]
```diff
@@ -195,80 +202,53 @@ source /home/tidb/.bash_profile
...
+    ```shell
+    tiup cluster template > topology.yaml
+    ```
 
-    以典型的 `两地三中心` 架构为例，介绍跨机房部署架构，以及需要注意的关键设置。
+    > **注意：**
+    >
+    > 混合部署场景也可以使用 `tiup cluster template --full > topology.yaml` 生成的建议拓扑模板，跨机房部署场景可以使用 `tiup cluster template --multi-dc > topology.yaml` 生成的建议拓扑模板。
+
+2. 配置集群拓扑文件
```
Comment (qiancai) [inline production-deployment-using-tiup.md:224]:
```suggestion
2. 配置集群拓扑文件。
```

File: production-deployment-using-tiup.md [hunk 16]
```diff
@@ -195,80 +202,53 @@ source /home/tidb/.bash_profile
...
+        - data_dir
+    - Server 参数
+        - PD servers
+        - TiDB servers
+        - TiKV servers
+        - Monitoring servers
+        - Grafana servers
+        - ......
+
+    下表列出了常用的 7 种场景，请根据链接中的拓扑说明以及配置文件模板新建一个配置文件`topology.yaml`。如果有其他组合场景的需求，请根据多个模板自行调整。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:241]:
```suggestion
    下表列出了常用的 7 种场景，请根据链接中的拓扑说明以及配置文件模板配置`topology.yaml`。如果有其他组合场景的需求，请根据多个模板自行调整。
```

File: production-deployment-using-tiup.md [hunk 17]
```diff
@@ -195,80 +202,53 @@ source /home/tidb/.bash_profile
...
+
+    | 场景 | 配置文件模板 | 拓扑说明 |
```
Comment (qiancai) [inline production-deployment-using-tiup.md:243]: 这个表格里拓扑说明列的内容更像是场景，而场景列里的内容更像是配置任务。

File: production-deployment-using-tiup.md [hunk 18]
```diff
@@ -278,50 +258,64 @@ alertmanager_servers:
...
 
 ## 第 4 步：执行部署命令
 
 > **注意：**
 >
 > 通过 TiUP 进行集群部署可以使用密钥或者交互密码方式来进行安全认证：
->
-> - 如果是密钥方式，可以通过 `-i` 或者 `--identity_file` 来指定密钥的路径；
-> - 如果是密码方式，可以通过 `-p` 进入密码交互窗口；
+
```
Comment (qiancai) [inline production-deployment-using-tiup.md:271]: 补充 > 否则注意左边的竖线会不连贯。

**Review by qiancai — APPROVED** [review]

**Pull request merged**